### DISCUSSION TITLE
Support for UESRPG d100 system

### DISCUSSION THREAD

**Discussion by 95Gman** [mclemente/fvtt-module-polyglot#153] [Ideas]
Currently the system has no support for languages.
https://gitlab.com/DogBoneZone/uesrpg-3e

List of languages are as follows: Cyrodilic, Ta'Agra, Nordic, Aldmeri, Jel, Dunmeri, Yoku, Bosmeri, Dwemeris, Falmer, Daedric, Ayleidoon, Dovah
Cyrodilic being the default.

**Comment by mclemente** [comment]
Hm, I might be able to implement it in a hacky way by using the Skill List as a placeholder for the languages.

**Comment by mclemente** [comment]
Try this version: [polyglot.zip](https://github.com/League-of-Foundry-Developers/fvtt-module-polyglot/files/6780511/polyglot.zip). Add skills with this pattern: `Language (Cyrodilic)`.
I have no idea about fonts for each language, I just put every elf language with the same font. There are some extra fonts on the `fonts` folder, tell me if you have a better idea on which font should be on each language.

**Reply by 95Gman** [reply]
That works, cheers.
https://www.nexusmods.com/skyrim/mods/88616 - Alyeid font (Just need to make a nexus account if you don't have one)
https://www.dafont.com/dovahkiin.font - Dovah font
https://www.dafont.com/daedra.font - Daedric font
https://www.fontspace.com/category/runic - Any of these for Nordic font
Rename 'Drowic' to 'Dunmeri'
Yoku is essentially Japanese.
https://www.dafont.com/mage-script.font - Bosmeri
https://www.dafont.com/alphabet-of-magi.font - Falmer

**Reply by mclemente** [reply]
I've released an update, check it out.
I couldn't use the Alyeid and Falmer fonts because they didn't include numbers, everything else was added.
